src/Files.App/ViewModels/ShellViewModel.cs

```diff
@@ -1956,7 +1960,7 @@ private void WatchForWin32FolderChanges(string? folderPath)
 				watcher.Deleted += DirectoryWatcher_Changed;
 				watcher.Renamed += DirectoryWatcher_Changed;
 				watcher.EnableRaisingEvents = true;
-			}
+			});
```

Comment:
```suggestion
			}, App.Logger);
```
Please set up the logger for debugging.